cliquet/resource.py

```diff
@@ -8,7 +8,7 @@
                                     HTTPNotFound, HTTPConflict)
 import six
 
-from cliquet import logger
+from cliquet.logger import *
```

Comment:
Nope was doing a test to understand why I was getting a couldn't find logger kind of error... :)